.env.example

```
@@ -0,0 +1,2 @@
+VITE_KAKAO_MAP_JS_KEY=your_key
+VITE_OPEN_WEATHER_API_KEY=your_key
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)